src/Evolution/VariableFixing/ParameterizedDeleptonization.cpp

```diff
@@ -122,6 +124,15 @@ void ParameterizedDeleptonization::correct_electron_fraction(
         get(equation_of_state.pressure_from_density_and_energy(
```

Comment:
In the 1d and 2d case, is there even an EOS call needed? And why don't you need to adjust pressure in the 3d case? If I remember correctly, ultimately you want to keep entropy constant and change Y_e, which would mean pressure, epsilon, and T change. Here it looks like you're wanting to fix epsilon and then vary T and p